contrib/cirrus/rootless_test.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+source $HOME/.bash_profile
+
+cd $GOSRC
+source $(dirname $0)/lib.sh
+
+req_env_var "
+GOSRC $GOSRC
+OS_RELEASE_ID $OS_RELEASE_ID
+OS_RELEASE_VER $OS_RELEASE_VER
+"
+
+echo "."
+echo "Hello, my name is $USER and I live in $PWD can I be your friend?"
+
+record_timestamp "rootless test start"
+
+set -x
+cd "$GOSRC"
+case "${OS_RELEASE_ID}-${OS_RELEASE_VER}" in
+    ubuntu-18) ;&  # Continue to the next item
+    fedora-29) ;&
+    fedora-28) ;&
+    centos-7) ;&
```

Comment:
we can skip centos-7 and rhel-7 for now as rootless doesn't work there

Reply:
oh okay, that's all just copy-paste.  Initially I was going to only run this vs F29, but it's easy to expand it later as needed.

Reply:
okay, got it.